doc/api/net.markdown

```diff
@@ -672,7 +674,10 @@ To listen on the socket `/tmp/echo.sock` the third line from the last would
 just be changed to
 
 ```js
-server.listen('/tmp/echo.sock', () => { /* 'listening' listener*/ })
+server.listen('/tmp/echo.sock', (err) => { 
+  /* 'listening' listener*/ 
```

Comment:
While you're here, could you fix both comments here and above to read be single line comments?

```js
// 'listening' listener
```